av1an-core/src/progress_bar.rs

```diff
@@ -153,10 +161,41 @@ pub fn dec_mp_bar(dec: u64) {
   }
 }
 
+pub fn update_mp_bar_info(kbps: f64, est_mb: f64) {
+  if let Some((_, pbs)) = MULTI_PROGRESS_BAR.get() {
+    pbs
+      .last()
+      .unwrap()
+      .set_message(format!(", {:.1} Kbps, est. {:.1} MB", kbps, est_mb));
```

Comment:
indicatif seems to have a wrapper for displaying bytes in a more human readable way: https://docs.rs/indicatif/0.16.2/indicatif/struct.HumanBytes.html
Perhaps we could just use that (or something similar) instead of just always displaying it in megabytes?